src/js/create.js

```diff
@@ -0,0 +1,105 @@
+    <input type="radio" class="btn-check" name="btnradio" id="btnradio1" autocomplete="off" checked>
+    <label class="btn btn-outline-primary" for="btnradio1">&#8451;</label>
+    <input type="radio" class="btn-check" name="btnradio" id="btnradio2" autocomplete="off">
+    <label class="btn btn-outline-primary" for="btnradio2">&#8457;</label>`;
+  header.appendChild(metric);
+  // search input
+  const search = document.createElement('div');
+  search.setAttribute('class', 'input-group');
+  search.innerHTML = `
+    <input type="text" class="form-control" placeholder="Search city">
+    <div class="input-group-append">
+      <button class="btn btn-secondary" type="button">
+        <i class="fa fa-search"></i>
+      </button>
+    </div>`;
+  header.appendChild(search);
+  return header;
+}
+
+function mainComponent() {
+  const main = document.createElement('main');
+  // section Weather
+  const weather = document.createElement('section');
+  weather.setAttribute('class', 'weather');
+  // location inside Weather section
+  const location = document.createElement('div');
+  location.setAttribute('class', 'location');
+  location.innerHTML = `
+    <h1><span id="city"></span>, <span id="country"></span></h1>
+    <p><span id="date"></span><span id="time"></span></p>
+    <div>
+      <p class="temp"><span id="temp"></span>&#8451;</p>
+      <figure><img id="icon" alt="Weather icon"></figure>
+    </div>
+    <div class="desc"><span id="desc"></span></div>`;
+  weather.appendChild(location);
+  // overcast inside Weather section
+  const overcast = document.createElement('div');
+  overcast.setAttribute('class', 'overcast');
+  overcast.innerHTML = `
+    <h4>Overcast</h4>
+    <p>Feels like: <span id="feels-like"></span>&#8451;</p>
+    <p>Wind: <span id="wind"></span>m/s</p>
+    <p>Humidity: <span id="humidity"></span>%</p>`;
+  weather.appendChild(overcast);
+  main.appendChild(weather);
+  // section Forecast
+  const forecast = document.createElement('section');
+  forecast.setAttribute('class', 'forecast');
+  forecast.innerHTML = `
+    <h4>3 days forecast</h4>
+    <div>
+      <span id="day1"></span>
+      <p><span id='temp-forecast-day1'></span>&#8451;</p>
+      <figure><img id="icon-day1" alt="Weather icon"></figure>
+    </div>
+    <div>
+      <span id="day2"></span>
+      <p><span id='temp-forecast-day2'></span>&#8451;</p>
+      <figure><img id="icon-day2" alt="Weather icon"></figure>
+    </div>
+    <div>
+      <span id="day3"></span>
+      <p><span id='temp-forecast-day3'></span>&#8451;</p>
+      <figure><img id="icon-day3" alt="Weather icon"></figure>
+    </div>`;
+  main.appendChild(forecast);
+  // section map
+  const map = document.createElement('section');
+  map.setAttribute('class', 'map-block');
+  map.innerHTML = `
+    <div id="map"></div>
+    <div>Latitude: <span id="latitude"></span></div>
+    <div>Longitude: <span id="longitude"></span></div>`;
+  main.appendChild(map);
+  return main;
+}
```

Comment:
[Clean code]
Here it is better to separate this from this file into "MainComponent" (if u want to have it named as Main).
And split it from the header 'component'